distribution/tools/plugin-cli/src/main/java/org/opensearch/plugins/InstallPluginCommand.java

```diff
@@ -341,38 +332,31 @@ private Path download(Terminal terminal, String pluginId, Path tmpDir, boolean i
         return downloadZip(terminal, pluginId, tmpDir, isBatch);
     }
 
-    // pkg private so tests can override
-    String getStagingHash() {
-        return System.getProperty(PROPERTY_STAGING_ID);
-    }
-
     boolean isSnapshot() {
         return Build.CURRENT.isSnapshot();
     }
 
     /** Returns the url for an official opensearch plugin. */
     private String getOpenSearchUrl(
         final Terminal terminal,
-        final String stagingHash,
         final Version version,
         final boolean isSnapshot,
         final String pluginId,
         final String platform
     ) throws IOException, UserException {
         final String baseUrl;
-        if (isSnapshot && stagingHash == null) {
+        if (isSnapshot == false && version.isRelease() == false) {
```

Comment:
why the `== false` & `== true`?! 😨

```suggestion
        if (!isSnapshot && !version.isRelease()) {
```

Reply:
This is the convention we use across the whole codebase

Reply:
oh, i hadn't noticed that yet! still appalled at it, though 🙈

Reply:
@rursprung It's even called out in the [developer guide](https://github.com/opensearch-project/OpenSearch/blob/034bd2b6483c180b4a4439d62452cc50198c37fb/DEVELOPER_GUIDE.md?plain=1#L353)! (I'm equally appalled though for the record)